src/backend/metal/src/command.rs

```diff
@@ -2385,21 +2583,69 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
 
     fn push_graphics_constants(
         &mut self,
-        _layout: &native::PipelineLayout,
-        _stages: pso::ShaderStageFlags,
-        _offset: u32,
-        _constants: &[u32],
+        layout: &native::PipelineLayout,
+        stages: pso::ShaderStageFlags,
+        offset: u32,
+        constants: &[u32],
     ) {
-        unimplemented!()
+        {
+            assert_eq!(offset & 3, 0);
```

Comment:
Maybe `ALIGN_MASK` like we did elsewhere instead of `3`

Reply:
sure!